# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/modules/desktop_capture/screen_drawer_linux.cc
# Commit: f9a9da6129c0
# Full Hash: f9a9da6129c01b5cefdc2ba2871d5bfe9fd0f0ab
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2019-04-17 21:47:29
# Regressor Bug: 1496359
# File Overlap Count: 1
# Description:
#   Bug 1496359 - Conflict resolutions for Pipewire support patch; r=ng
#   
#   Depends on D27368
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27369
# ==============================================================================

diff -r 3f0981f37c1c -r f9a9da6129c0 media/webrtc/trunk/webrtc/modules/desktop_capture/screen_drawer_linux.cc
--- a/media/webrtc/trunk/webrtc/modules/desktop_capture/screen_drawer_linux.cc	Mon Apr 15 17:26:27 2019 +0000
+++ b/media/webrtc/trunk/webrtc/modules/desktop_capture/screen_drawer_linux.cc	Mon Apr 15 17:31:02 2019 +0000
@@ -13,9 +13,9 @@
 
 #include <memory>
 
+#include "modules/desktop_capture/linux/shared_x_display.h"
 #include "modules/desktop_capture/screen_drawer.h"
 #include "modules/desktop_capture/screen_drawer_lock_posix.h"
-#include "modules/desktop_capture/x11/shared_x_display.h"
 #include "rtc_base/checks.h"
 #include "rtc_base/ptr_util.h"
 #include "system_wrappers/include/sleep.h"